app/assets/stylesheets/code.scss

```diff
@@ -5,3 +5,12 @@ pre {
   white-space: pre;
   code { white-space: pre; }
 }
+
+// For inline-code inside links, let's force the color to the default link
+// color to make them be recognizable as links.
+p a {
+  code,
+  pre {
```

Comment:
With `pre` it would even work for multi-line code blocks. If you only want inline-code blocks, then `code` would be enough, but I guess it doesn't hurt? Or do we actually use `pre` somewhere else than in multi-line code blocks?

Reply:
This does **not** actually work:

```md
[<pre>foo
bar</pre>](https://example.com)
```

It looks like it works, but it doesn't. The markdown parser will transform that into


```html
<pre><a href="https://example.com" target="_blank" rel="noopener noreferrer">foo<br>
bar</a></pre>
```

So there isn't actually a `<pre>` **inside** a link. My motivation for adding `pre` as well is to adequately override where the original color comes from (Bootstrap, actually). We don't do pre-inside-a, it would be invalid, but Bootstrap supports it, I overrode that. And, well, because an extra selector adds virtually no cost to anything. :)